### DISCUSSION TITLE
Rotary substitution

### DISCUSSION THREAD

**Discussion by pasmoi87** [keirf/flashfloppy#709] [Ideas]
Hi,
Is it possible to use EVQWKA001 as Rotary ? 
https://www.farnell.com/datasheets/1679881.pdf
I am looking for a scroll wheel,  EVQWKA00 has 15 detents versus 20 for Rotary EC11 and switch is present.
But I don't find anything on pinout (CLK, DT, SW, GND) of EVQWKA001. Mystery ?
I find also  SIQ-02FVC3 SIQ-02FVC3, SIQ-02FVC1 or SIQ-02FVS3 but without switch and no pinout too.
It will be great if it was possible to use it.
Thanks. Cordialy.

**Comment by keirf** [comment]
Connect as follows: Note that A/B can be swapped. Also SW1/SW2 an be swapped. 
CLK = A, DAT = B (or vice versa, if the direction ends up reverse of what you want!)
SW = SW2
GND = COM, SW1

**Reply by pasmoi87** [reply]
Well seen. I had not seing B, SW1 etc... on the plan.
OK thanks. A and B are CLK or DAT. I will test, this scroll wheel is what I am looking. No problem for the switch, it's logic.

**Reply by keirf** [reply]
A, B, C are actually the more standard names for rotary pins. Probably I should mention them in the wiki.
